source/uri-options/tests/proxy-options.json

```diff
@@ -0,0 +1,58 @@
+{
+  "tests": [
```

Comment:
We should have some tests with valid proxy configuration as well, not just invalid ones.

Additionally consider adding tests to validate that multiple occurrences of `proxyHost` or other option are invalid.

Reply:
I would like to see valid URI configurations tested as well. It feels wrong to me to describe all the invalid configurations of a new option without describing the valid ones.

In terms of implementing features... As long as the SOCKS5 proxy tests are isolated in their own json/yml file (which they are), then drivers can (and should) ignore them if they're not yet implementing proxy support. The .NET/C# driver loads all json files in a test directory but has a list of excluded test filenames for features we haven't implemented yet. This shows clear intent that we haven't done a particular piece of work rather than wondering if we forgot to sync a particular test file.

Reply:
Sure, done!